go/signedexchange/cmd/dump-signedexchange/main.go

```diff
@@ -57,12 +67,12 @@ func verify(e *signedexchange.Exchange) error {
 	if *flagCert != "" {
```

Comment:
would you factor this part out so jsonPrintHeaders can also use `*flagCert` if available?

I think we can create a `func initCertFetcher() (func(url string) ([]byte, error), error)` which returns signedexchange.DefaultCertFetcher if no `*flagCert` and read `*flagCert` if avail.

Reply:
Nice, done!